kratos/processes/assign_scalar_field_to_conditions_process.h

```diff
@@ -36,21 +36,21 @@ namespace Kratos
 /// The base class for assigning a value to scalar variables or array_1d components processes in Kratos.
 /** This function assigns a value to a variable belonging to all of the nodes in a given mesh
 */
-class AssignScalarFieldToConditionsProcess : public Process
```

Comment:
this shouldn't be changed

Reply:
This has conflicts wit the solid app, you copied and changed the behaviour without changing the name. It is used only from python, so no change in behaviour has to appear